business_requirement_sale/models/sale_order.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2019 Tecnativa - Victor M.M. Torres
+# Copyright 2019 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    business_requirement_id = fields.Many2one(
+        comodel_name="business.requirement", string="Business requirement"
+    )
+
+
+class SaleOrderLine(models.Model):
```

Comment:
Move this class code to the sale_order_line file since it will enhance readability